### PR TITLE
Add Scrapy `ApifyHttpProxyMiddleware` for managing proxies

### PR DESCRIPTION
**Pull request by vdusek**
### Issue

- Closes: https://github.com/apify/actor-templates/issues/255

### Release process

- After the SDK update (https://github.com/apify/apify-sdk-python/pull/158) is merged, release a new version, and use it here.
- Once we finish the review process, update the [scrapy wrapper](https://github.com/apify/actor-templates/tree/master/wrappers/python-scrapy) and [actor-scrapy-books-example](https://github.com/apify/actor-scrapy-books-example/).

### CODE REVIEW

**Review by janbuchar — APPROVED** [review]

File: templates/python-scrapy/.actor/actor.json [hunk 1]
```diff
@@ -8,35 +8,5 @@
         "templateId": "python-scrapy"
     },
     "input": "./input_schema.json",
-    "dockerfile": "./Dockerfile",
-    "storages": {
```
Comment (honzajavorek) [inline templates/python-scrapy/.actor/actor.json:12]: Why is this removed?
